### PR TITLE
Open new tab screen on browser action click

### PR DESCRIPTION
**Pull request by Temmyogunbo**
This PR implements chrome browser actions

### CODE REVIEW

File: packages/extension/src/background/browser-action.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Extension } from '@mujo/utils'
+
+const { tabs } = Extension
+
+export const onBrowserAction = () =>
+  tabs.create({ url: 'chrome://newtab' })
```
Comment (jcblw) [inline packages/extension/src/background/browser-action.js]: Can you see if you can open up the path to the index file so we do not need to up date this when we make new tab optional?

https://developer.chrome.com/extensions/runtime#method-getURL

**Comment by jcblw** [comment]
Everything looks good, just confirm for me this works once you reload your background page. I can test it tonight but overall great work dude.

File: packages/extension/src/background/browser-action.js (hunk 1 above)
Comment (Temmyogunbo) [reply packages/extension/src/background/browser-action.js]: Yes, it does open up a new tab.

File: packages/extension/src/background.js [hunk 2]
```diff
@@ -33,4 +34,7 @@ alarms.onAlarm.addListener(alarmReducer)
 onMessage(reducer)
 webNavigation.onCommitted.addListener(injectScript)
 
+// Called when the user clicks on the browser action.
+chrome.browserAction.onClicked.addListener(onBrowserAction)
```
[the remark this answers is not in the document]
Comment (jcblw) [reply packages/extension/src/background.js]: Ah you know what probably was happening now that I think about  it, is was that you need to rerun `npm run bootstrap` after updating the extension.js file. We need to use the extension file that is going to allow us to make a Firefox extension much easier.

File: packages/extension/src/background/browser-action.js (hunk 1 above)
Comment (jcblw) [reply packages/extension/src/background/browser-action.js]: Can you get this to open up a direct reference to the index.html file?

File: packages/extension/src/background.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Temmyogunbo) [reply packages/extension/src/background.js]: It worked. Thanks

**Comment by jcblw** [comment]
Dude looks great thanks for the revisions, but I am not seeing the browser action addition to the extension file on this pr. Was it removed? Also for future reference it's best to allow your got history to show here. Please do not squash, for instance it would be easy for me to help more with the browser action missing stuff if squashing was not happening. I spend some time to put together a contributing md soon.

Can you write some test for that background file but we merge this.

**Comment by Temmyogunbo** [comment]
I forgot to push it.

**Review by jcblw — APPROVED** [review]

**Comment by jcblw** [comment]
Thanks @Temmyogunbo I'll merge it in tonight.

**Comment by jcblw** [comment]
@Temmyogunbo just noticing, but it seems that checks are not running on your PR. Not sure exactly why but I can only assume its because of the **force pushing** that you are doing in this PR. Can you __please not do force pushing anymore__ and allow for a rich commit history of the changes in this pull request? 

I am not sure I should be using admin rights to merge this since the check did not run. Can you create an empty commit to test and see if that triggers the checks to work?

```
git commit -m "trigger Github checks" --allow-empty
git push origin task/implement-browser-action
```

talk to you tomorrow 😄

**Comment by jcblw** [comment]
https://github.com/mujo-code/source/commit/721eff5084807b7257b054f2feaeb153bc1fbc9b/checks?check_suite_id=269294628

Here is the post merge push action run

**Pull request merged**